tests/integration/security/fuzz/fuzz_test.go

```diff
@@ -99,10 +114,85 @@ func TestFuzzAuthorization(t *testing.T) {
 			deploy(t, apacheServer, ns, "backends/apache/apache.yaml")
 			deploy(t, nginxServer, ns, "backends/nginx/nginx.yaml")
 			deploy(t, dotdotpwn, ns, "fuzzers/dotdotpwn/dotdotpwn.yaml")
+			waitService(t, apacheServer, ns)
+			waitService(t, nginxServer, ns)
 			for _, target := range []string{apacheServer, nginxServer} {
 				t.NewSubTest(target).Run(func(t framework.TestContext) {
 					runDotdotPwnTest(t, ns, target)
 				})
 			}
 		})
 }
+
+func runJwtToolTest(t framework.TestContext, ns, server string) {
+	pods, err := t.Clusters().Default().PodsForSelector(context.TODO(), ns, "app="+jwtTool)
+	if err != nil {
+		t.Fatalf("failed to get jwttool pod: %v", err)
+	}
+	t.Logf("running jwttool fuzz test against the %s (should normally complete in 10 seconds)...", server)
+
+	// Run the jwttool fuzz testing with "--mode at" to run all tests:
+	// - JWT Attack Playbook
+	// - Fuzz existing claims to force errors
+	// - Fuzz common claims
+	commands := []string{
+		"./run.sh",
+		"--targeturl",
+		fmt.Sprintf("http://%s:8080/private/secret.html", server),
+		"--noproxy",
+		"--headers",
+		fmt.Sprintf("Authorization: Bearer %s", jwtToken),
```

Comment:
I thought we were fuzzing the jwt? so why is it hard coded?

Reply:
Is this like a seed or something?

Reply:
this is the base jwt token, the tool will mutate it to generate other tokens, also updated to use multiple different base tokens.